### PR TITLE
Apex predator & predator senses locker fixes

### PR DESCRIPTION
**Pull request by Adrer**
## What Does This PR Do
Apex predator and predator senses (Hemomancer power) no longer considers it a different Z-level when someone is in a locker.
Fixes: #25767

## Why It's Good For The Game
Should be able to smell someone just fine from a locker

## Testing
Be a changeling.
Smell someone who is inside a locker.
Smell someone while I am inside a locker.

## Changelog
:cl:
fix: Apex predator can now be used on mobs in a locker, and while you are in a locker.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/changeling/powers/apex_predator.dm [hunk 1]
```diff
@@ -9,8 +9,12 @@
 
 /datum/action/changeling/apex_predator/sting_action(mob/user)
 	var/list/target_by_name = list()
+	var/turf/target_turf
+	var/turf/user_turf
 	for(var/mob/living/carbon/human/possible_target as anything in GLOB.human_list)
-		if(!possible_target.mind || possible_target.z != user.z)
+		target_turf = get_turf(possible_target)
+		user_turf = get_turf(user)
```
Comment (DGamerL) [inline code/modules/antagonists/changeling/powers/apex_predator.dm:16]:
```suggestion
		var/turf/target_turf = get_turf(possible_target)
		var/turf/user_turf = get_turf(user)
```

File: code/modules/antagonists/changeling/powers/apex_predator.dm [hunk 2]
```diff
@@ -9,8 +9,12 @@
 
 /datum/action/changeling/apex_predator/sting_action(mob/user)
 	var/list/target_by_name = list()
+	var/turf/target_turf
+	var/turf/user_turf
```
Comment (DGamerL) [inline code/modules/antagonists/changeling/powers/apex_predator.dm:13]:
```suggestion
```

**Review by DGamerL — COMMENTED** [review]
Why not do this?

**Review by DGamerL — APPROVED** [review]

**Review by hal9000PR — APPROVED** [review]
oops

**Pull request merged**